### ISSUE TITLE
Documentation still claims Racket 3M is default

### ISSUE THREAD

**Issue by LiberalArtist** [racket/racket#3663]
The [documentation](https://pre-release.racket-lang.org/doc/guide/performance.html#%28part._virtual-machines%29) for the 7.9.0.900 release candidate still says that “3m is the current default implementation”. Should this be changed before the Racket 8 release?

https://github.com/racket/racket/blob/4e76091c6498ff470c0990eb6bfbaa6925424525/pkgs/racket-doc/scribblings/guide/performance.scrbl#L60-L61

**Comment by sorawee** [comment]
Similarly, in the doc of `raco exe` (https://docs.racket-lang.org/raco/exe.html)

```
 @item{@DFlag{3m} --- generate an executable based on the @gtech{3m}
   variant of Racket, which is the default unless running a @exec{raco
   exe} that is based on the @gtech{CGC} or @gtech{CS} variant.}

 @item{@DFlag{cgc} --- generate an executable based on the @gtech{CGC}
   variant of Racket, which is the default only when running a
   @exec{raco exe} that is based on the @gtech{CGC} variant.}

 @item{@DFlag{cs} --- generate an executable based on the @gtech{CS}
   variant of Racket, which is the default only when running a
   @exec{raco exe} that is based on the @gtech{CS} variant.}
```

It still sounds like 3m is the default.

**Comment by jbclements** [comment]
Thanks! I made your proposed edits as part of 

https://github.com/racket/racket/pull/3668

(assuming I didn’t mess it up….)

John

**Comment by sorawee** [comment]
Fixed with c5916b7b1dc2f2a33d15775daaf7ce4f1f49930a, 936ac2bf75deaf08ff6338f2de5cc37205163670, and a5b14d74b7f75e2386dd845413d2ac81c89e6c80. Closing.
